File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_B/f10p2_V5_bigint/w.kt
// F10.P2 kotlin WITNESS V5 (arbitrary precision: kotlin-stdlib declares operator plus on BigInteger).
import java.math.BigInteger
fun probe(i: BigInteger, d: BigInteger): BigInteger {
    // BEGIN PROBE F10.P2
    val sum = i + d
    // END PROBE F10.P2
    return sum
}
fun main() { println("sum=" + probe(BigInteger("9223372036854775807"), BigInteger.ONE) + " (exact, no overflow edge)") }

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/witnesses/F10.P2/go/c3/w.go
package main

import "fmt"

const i = 3

func probe() float64 {
	var d float64 = 0.5
// BEGIN PROBE F10.P2
sum := float64(i) + d
// END PROBE F10.P2
	return sum
}

func main() { fmt.Println(probe()) }
